resources/simulation.ts

```diff
@@ -0,0 +1,61 @@
+import { BaseResource } from ".";
+import { ReasonCode } from "../types";
+import { UnitConfig, UnitResponse } from "../types/common";
+import {
+  SimulateApproveApplicationRequest,
+  SimulateDenyApplicationRequest,
+  SimulateRejectDocumentRequest,
+} from "../types/simulation";
+
+export class Simulations extends BaseResource {
+  constructor(token: string, basePath: string, config?: UnitConfig) {
+    super(token, basePath + "/statements", config);
+  }
+
+  public async applicationApprove(
+    applicationId: string,
+    request: SimulateApproveApplicationRequest
+  ): Promise<UnitResponse<unknown>> {
+    return this.httpPost<UnitResponse<unknown>>(
```

Comment:
Can you resolve the type here from `unknown` to the type of the response? Same goes for the rest of the endpoints here.